src/entries/background/procedures/popup/index.ts

```diff
@@ -9,7 +11,16 @@ import { stateRouter } from './state';
 import { telemetryRouter } from './telemetry';
 import { walletRouter } from './wallet';
 
-const sentryMiddleware = os.middleware(async ({ next }) => {
+const sentryMiddleware = os.middleware(async ({ next, path }) => {
+  if (INTERNAL_BUILD || IS_TESTING) {
+    Sentry.addBreadcrumb({
+      message: `ORPC: ${path.join('/')}`,
+      data: {
+        path: path.join('/'),
+        timestamp: new Date().toISOString(),
+      },
+    });
```

Comment:
Any reason we don't want to expose this for all releases?

Reply:
I think it's too much data, and it will be pretty repetitive
lets see, we can always keep it